src/saleor_app/install.py

```diff
@@ -18,7 +18,7 @@ async def install_app(
     saleor_domain: DomainName,
     auth_token: AppToken,
     manifest: Manifest,
-    events: Dict[str, SaleorEventType],
+    events: Dict[str, tuple[SaleorEventType, str]],
```

Comment:
using `tuple` instead of `Tuple` from typing will raise Python version requirement.